packages/main/cypress/specs/TimePicker.cy.tsx

```diff
@@ -0,0 +1,385 @@
+import "../../src/Assets.js";
+import { setLanguage } from "@ui5/webcomponents-base/dist/config/Language.js";
+import TimePicker from "../../src/TimePicker.js";
+import Label from "../../src/Label.js";
+
+function pressKeyNTimes(key: "ArrowDown" | "ArrowUp" | "Space" | "Tab" | "Enter", n: number) {
+	for (let i = 0; i < n; i++) {
+		cy.realPress(key);
+	}
+}
+
+describe("TimePicker Tests", () => {
+	it("input receives value in format pattern depending on the set language", () => {
+		cy.wrap({ setLanguage })
+		.then(api => {
+			return api.setLanguage("bg")
+		})
+
+		cy.mount(<TimePicker value="03:16:16"></TimePicker>);
+
+		cy.get("[ui5-time-picker]")
+			.as("timePicker");
+
+		cy.get<TimePicker>("@timePicker")
+			.ui5TimePickerGetInnerInput()
+			.should("have.value", "3:16:16 ч.");
+
+		cy.wrap({ setLanguage })
+			.invoke("setLanguage", "en");
+	});
+
+	it("tests clocks value", () => {
+		cy.mount(<TimePicker formatPattern="HH:mm:ss" value="11:12:13"></TimePicker>);
+
+		cy.get("[ui5-time-picker]")
+			.as("timePicker")
+			.ui5TimePickerValueHelpIconPress();
+
+		cy.get<TimePicker>("@timePicker")
+			.ui5TimePickerGetClock("hours")
+			.should("have.prop", "valueNow", 11);
+
+		cy.get<TimePicker>("@timePicker")
+			.ui5TimePickerGetClock("minutes")
+			.should("have.prop", "valueNow", 12);
+
+		cy.get<TimePicker>("@timePicker")
+			.ui5TimePickerGetClock("seconds")
+			.should("have.prop", "valueNow", 13);
+	});
+
+	it("tests clocks submit value", () => {
+		cy.mount(<TimePicker formatPattern="hh:mm:ss" value="12:00:01"></TimePicker>);
+
+		cy.get("[ui5-time-picker]")
+			.as("timePicker")
+			.ui5TimePickerValueHelpIconPress();
+
+		cy.get("@timePicker")
+			.ui5TimePickerGetClock("hours")
+			.realClick()
+			.should("be.focused")
+
+
+		pressKeyNTimes("ArrowDown", 10);
+		cy.realPress("Space");
+
+		cy.get("@timePicker")
+			.ui5TimePickerGetClock("minutes");
+
+		pressKeyNTimes("ArrowDown", 22);
```

Comment:
do we need to click that much times the arrow keys? is there some specific behavior which is testsed? can we reduce to 1-2 times ?